redirects.yaml

```diff
@@ -465,6 +465,18 @@ redirects:
 - from: /docs/extensions/declarativenetrequest
   to: /docs/extensions/reference/declarativeNetRequest
 
+- from: /docs/extensions/reference/declarativeNetRequest#implementation
```

Comment:
Do redirects work for hash tags?

Reply:
You can use an anchor redirect
https://developer.chrome.com/docs/handbook/how-to/create-anchor-redirects/

Reply:
I can't work out from those instructions what I'm supposed to do.

Reply:
Sweet, I wasn't aware that we support anchor redirects. The yaml file defines server-side redirects which are not supported for anchor links. Hence the client-side based workaround which needs to be defined in the page frontmatter itself:

```
---
...
anchorRedirects:
  implementation: /docs/extensions/reference/declarativeNetRequest#implementation-web-accessible-resources
---
```

Reply:
A redirect needs two targets. This only has one.

Reply:
<img width="743" alt="image" src="https://github.com/GoogleChrome/developer.chrome.com/assets/380472/9af1304a-7fac-485b-a74b-4b2fe6bcf389">

Reply:
I would expect that to only work when the anchor name is the same. I reused anchor names wherever I could. But there are several cases where it just made no sense.

Reply:
Plus how would the server know to look in /recorder/reference for a redirect of something in recorder/?

Reply:
This is handled client side afaik.